duopull/README.md

```diff
@@ -0,0 +1,52 @@
+# Cloud Function for Duo Log Collection
+
+This is a Cloud function that pulls administration/authentication/telephony logs
+from the Duo API and pushes them into a logging pipeline via a Kinesis stream.
```

Comment:
There are a bunch of references to Kinesis in the documentation and also in comments throughout the code; we should probably change these to reflect Stackdriver use?